deploy/template.yaml

```diff
@@ -35,8 +31,8 @@ Conditions:
   IsDevelopment: !Or
     - !Equals [ !Ref AWS::AccountId, "130355686670"]
     - !Equals [ !Ref AWS::AccountId, "175872367215"]
-  IsOldDevelopment: !Equals [ !Ref AWS::AccountId, "130355686670"]
-  IsNewDevelopment: !Equals [ !Ref AWS::AccountId, "175872367215"]
+  IsDev01: !Equals [ !Ref AWS::AccountId, "130355686670"]
+  IsDev02: !Equals [ !Ref AWS::AccountId, "175872367215"]
```

Comment:
FYI another property might need changing after my latest merge, as it's also used to control the `ANALYTICS_DOMAIN` (see: https://github.com/alphagov/di-ipv-core-front/pull/1051/files#r1317427584)

Let me know if you want to discuss :)

Reply:
Hi Joe - you will need to change your commit as relevant - this allows for devs to deploy to either dev01 or dev02 dev accounts.

Reply:
Sorry, mine was just merged... you should see it if you rebase.

It looks like the domains will now be `xxx.01.dev.identity.account.gov.uk` or `xxx.02.dev.identity.account.gov.uk`? - in which case the analytics domain in dev will need to match (maybe we can pull out a variable or something for the domain to make it easier).

Let me know if I can help

Reply:
@Joe-Edwards-GDS changes are to make the dev analytics domain follow the previously correct pattern, the .01 or .02 sub(subsub...?) domain will need to be included as well now

Reply:
Ok - do you want me to update this branch accordingly? I think you'll need to rebase to see my changes.

Reply:
Joe I am going to merge this one in feel free to open this branch up again or use a different branch completely up to you - need this in to start testing dev deployments.